README.md

```diff
@@ -23,9 +23,10 @@ sandbox commands:
   copyFrom <file> -> copy <file> from the algod container. Useful for offline transactions & LogicSigs plus TEAL work.
 
 algorand commands:
-  logs        -> stream algorand logs with the carpenter utility.
-  status      -> get node status.
-  goal (args) -> run goal command like 'goal node status'.
+  logs            -> stream algorand logs with the carpenter utility.
+  status          -> get node status.
+  goal (args)     -> run goal command like 'goal node status'.
+  tealdbg (args)  -> run tealdbg command to debug program execution. Always use `tealdbg` with `--listen 0.0.0.0` or `--listen [IP ADDRESS]` falgs, if you need to acess tealdbg from outside of algod docker container.
```

Comment:
Please relocate this to the README.md file:
"Always use `tealdbg` with `--listen 0.0.0.0` or `--listen [IP ADDRESS]` falgs, if you need to acess tealdbg from outside of algod docker container."